tests/utils/azure_users_test.py

```diff
@@ -0,0 +1,44 @@
+from unittest.mock import Mock, patch
+from utils.azure_users import AzureConnection, ROLE_FIELD_VALUES, AzureUser_v2
+from pytest import mark
+
+
+@patch('msal.ConfidentialClientApplication')
+@patch('utils.azure_users.AzureConnection.get_token')
+@patch('utils.azure_users.AzureConnection._get_user')
+@mark.parametrize(
+    'field_name,field_value,expected',
```

Comment:
Maybe we could change "expected" to "expectedTestUser"

Reply:
Sure, we have changed to `is_test_user_expected_value` it's a more descriptive name.